securitycenter/bigqueryv2/bigquery_test.go

```diff
@@ -0,0 +1,297 @@
+		err = getBigQueryExport(buf, parent, configID)
+
+		if err != nil {
+			r.Errorf("getBigQueryExport() had error: %v", err)
+			return
+		}
+
+		got := buf.String()
+
+		fmt.Println(got)
+
+		if !strings.Contains(got, configID) {
+			r.Errorf("getBigQueryExport() got: %s want %s", got, configID)
+		}
+
+		// Cleanup
+		cleanupBigQueryExport(t, configID)
+	})
+}
+
+func TestDeleteBigQuery(t *testing.T) {
+
+	testutil.Retry(t, 5, 20*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+		// Create Test BigQueryExport Config
+		rand, err := uuid.NewUUID()
+		if err != nil {
+			r.Errorf("Issue generating id.")
+			return
+		}
+		configID := "random-bqexport-id-" + rand.String()
+
+		if err := addBigQueryExport(t, configID); err != nil {
+			r.Errorf("Could not setup test environment: %v", err)
+			return
+		}
+
+		parent := fmt.Sprintf("organizations/%s/locations/global", orgID(t))
+
+		// Call DeleteBigQueryExport
+		err = deleteBigQueryExport(buf, parent, configID)
+
+		if err != nil {
+			r.Errorf("getBigQueryExport() had error: %v", err)
+			return
+		}
+
+		got := buf.String()
+
+		fmt.Println(got)
+
+		if !strings.Contains(got, configID) {
+			r.Errorf("deleteBigQueryExport() got: %s want %s", got, configID)
+		}
+	})
+}
+
+func TestUpdateBigQuery(t *testing.T) {
+
+	testutil.Retry(t, 5, 20*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+
+		// Create Test BigQueryExport Config
+		rand, err := uuid.NewUUID()
+		if err != nil {
+			r.Errorf("Issue generating id.")
+			return
+		}
+		configID := "random-bqexport-id-" + rand.String()
+
+		if err := addBigQueryExport(t, configID); err != nil {
+			r.Errorf("Could not setup test environment: %v", err)
+			return
+		}
+
+		parent := fmt.Sprintf("organizations/%s/locations/global", orgID(t))
+
+		// Call UpdateBigQueryExport
+		err = updateBigQueryExport(buf, parent, configID)
+
+		if err != nil {
+			r.Errorf("getBigQueryExport() had error: %v", err)
+			return
+		}
+
+		got := buf.String()
+
+		fmt.Println(got)
+
+		if !strings.Contains(got, configID) {
+			r.Errorf("getBigQueryExport() got: %s want %s", got, configID)
+		}
+
+		// Cleanup
+		cleanupBigQueryExport(t, configID)
+	})
+}
+
+func TestCreateBigQuery(t *testing.T) {
+
+	testutil.Retry(t, 5, 20*time.Second, func(r *testutil.R) {
+		buf := new(bytes.Buffer)
+
+		rand, err := uuid.NewUUID()
+		if err != nil {
+			r.Errorf("Issue generating id.")
+			return
+		}
+		configID := "random-bqexport-id-" + rand.String()
+
+		parent := fmt.Sprintf("organizations/%s/locations/global", orgID(t))
+
+		err = createBigQueryExport(buf, parent, configID, bigQueryDatasetName(t))
+
+		if err != nil {
+			r.Errorf("getBigQueryExport() had error: %v", err)
+			return
+		}
+
+		got := buf.String()
+
+		fmt.Println(got)
+
+		if !strings.Contains(got, configID) {
+			r.Errorf("createBigQueryExport() got: %s want %s", got, configID)
+		}
+
+		// Cleanup
+		cleanupBigQueryExport(t, configID)
+
+	})
+}
+
+func TestMain(m *testing.M) {
```

Comment:
This TestMain method is unnecessary when you're not doing any additional work besides `os.Exit(m.Run)` - the framework will do this for you.